draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -272,13 +272,23 @@ and using spare capacity is discussed in Sections 6, 11.1, and 12
 of {{?RFC3649}}.
 
 (2)
+: Wireless links
+
+: While the early internet was dominated by wired links, the properties
+of wireless links have become extremely important to internet performance.
+In particular, congestion controllers should be evaluated in situations
+where some packet losses are due to radio effects, rather than router
+queue drops; the bandwidth varies over time due to changing link conditions;
+and media access delays and link-layer retransmission lead to increased jitter
+in round-trip times. See {{?RFC3819} and Section 16 of {{Tools}} for further
+discussion of wireless properties.
+
+(3)
 : Difficult Environments.
 
 : The proposed algorithms should be assessed in difficult
-environments such as paths containing wireless links.
-Characteristics of wireless environments are discussed in
-{{?RFC3819}} and in Section 16 of {{Tools}}.  Other difficult
-environments can include those with multipath routing within a
+environments 
+such as those with multipath routing within a
```

Comment:
multipath routing - do you mean methods like ECMP or something else? - I was unsure.